### DISCUSSION TITLE
Are the samples capable of transmitting dats over different networks?

### DISCUSSION THREAD

**Discussion by Silhim** [eclipse-edc/Samples#16] [Q&A]
Hello dear EDC team,
Im currently getting my hands on the edc samples and Im playing around with it a little bit. I wondering if the samples are capable of transmitting data over two different networks. I'm trying it with the transfer-01 sample. I've adjusted the IP addresses in the config.properties, in the filetransfer.json, in the contractoffer.json and in the commands. Im getting to the point where the request to the file is being send but then the provider is sending the file to itself and the connector is never receiving it.
Any thoughts on this? I've checked the paths tho.
Thanks a lot in advance and best regards

**Comment by ndr-brt** [comment]
The transfer-01 transfers a file using the file system, so it's not the best sample to be used over different networks. I'd suggest to take a look at this PR, that will introduce a transfer through http using data-plane: https://github.com/eclipse-edc/Samples/pull/8
